core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.spec.ts

```diff
@@ -65,6 +71,23 @@ describe('Topic editor tab directive', function() {
     }
   };
 
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [
+        TopicEditorRoutingService,
+        {provide: Location, useClass: SpyLocation}]
+    });
+  });
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    for (let servicePair of angularServices) {
+      $provide.value(
+        servicePair[0], TestBed.get(servicePair[1]));
```

Comment:
This is the same code as the one in `importAllAngularServices` function but with a slight modification, so he had to use this instead of `importAllAngularServices()`